### PR TITLE
Returning event object upon event update

### PR DESCRIPTION
**Pull request by viniciusvieir**

### CODE REVIEW

File: server/src/controllers/event.js [hunk 1]
```diff
@@ -81,7 +81,9 @@ class EventController {
       { title, description, date, duration_min, max_participants }
     )
 
-    res.json({ success: true })
+    const event = await Event.findById(eventId)
+
+    res.json(event)
```
Comment (NeelSalvi) [inline server/src/controllers/event.js:88]: Look grand, Thanks

**Review by NeelSalvi — CHANGES_REQUESTED** [review]

File: server/src/controllers/event.js (hunk 1 above)
Comment (NeelSalvi) [reply server/src/controllers/event.js:88]: Can the number of the active events be sent on the trail object?
Also, a route for viewing events based on userID is required and a route for joining events.

Comment (viniciusvieir) [reply server/src/controllers/event.js:88]: Added: 

- Outing count on the route /trails. Use as `/trails?fields=outing_count`
- Join event: make a post request to `/trails/:trailId/events/:eventId/join`

**Comment by NeelSalvi** [comment]
Reviewed, Mearging

**Pull request merged**